src/components/common/buttons/CustomButton.js

```diff
@@ -80,6 +117,12 @@ const styles = StyleSheet.create({
   buttonWrapperText: {
     minHeight: 34,
     justifyContent: 'center'
+  },
+  leftIcon: {
+    marginRight: normalize(10)
```

Comment:
the number looks wrong not multiple of 8

Reply:
Done